x/auth/vesting/client/cli/tx.go

```diff
@@ -44,7 +45,7 @@ func NewMsgCreateVestingAccountCmd() *cobra.Command {
 account can either be a delayed or continuous vesting account, which is determined
 by the '--delayed' flag. All vesting accouts created will have their start time
 set by the committed block's time. The end_time must be provided as a UNIX epoch
-timestamp.`,
+timestamp. You can also optionally configure the 'admin' field using the flag '--admin {addr}. This admin will be able to perform some administrative actions on the vesting account if set.`,
```

Comment:
Looks like the vesting submodule doesn't have a query handler, is there a way to query or display the account admin from the CLI/RPC endpoint or add it a query handler or display it in `seid q auth account(s)`?

Reply:
it's returned when querying the actual account `seid q auth account $addr`